edagames/settings.py

```diff
@@ -125,3 +137,9 @@
 LOGOUT_REDIRECT_URL = '/login'
 STATICFILES_DIRS = ['static']
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+LOGIN_URL = 'login'
+LOGOUT_URL = 'logout'
+
+
+SOCIAL_AUTH_FACEBOOK_KEY = '1194039207691875'  # App ID
+SOCIAL_AUTH_FACEBOOK_SECRET = 'f469bcead34a7766f1fa01f81b040ddf'  # App Secret
```

Comment:
Is there a way to use an `ENV_VARIABLE` instead of hardcoding these keys? And how you're going to manage these secrets along different environments?